dotnetv3/Lambda/scenarios/Lambda_Basics/Lambda_Basics/LambdaMethods.cs

```diff
@@ -0,0 +1,205 @@
+            var functionList = reponse.Functions;
+            return functionList;
+        }
+
+        // snippet-end:[lambda.dotnetv3.Lambda_Basics.ListFunctions]
+
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.GetFunction]
+
+        /// <summary>
+        /// Gets information about a Lambda function.
+        /// </summary>
+        /// <param name="client">The initialized Lambda client object.</param>
+        /// <param name="functionName">The name of the Lambda function for
+        /// which to retrieve information.</param>
+        /// <returns>A System Threading Task.</returns>
+        public async Task<FunctionConfiguration> GetFunction(AmazonLambdaClient client, string functionName)
+        {
+            var functionRequest = new GetFunctionRequest
+            {
+                FunctionName = functionName,
+            };
+
+            var response = await client.GetFunctionAsync(functionRequest);
+            return response.Configuration;
+        }
+
+        // snippet-end:[lambda.dotnetv3.Lambda_Basics.GetFunction]
+
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.CreateLambdaFunction]
+
+        /// <summary>
+        /// Creates a new Lambda function.
+        /// </summary>
+        /// <param name="client">The initialized Lambda client object.</param>
+        /// <param name="functionName">The name of the function.</param>
+        /// <param name="s3Bucket">The S3 bucket where the zip file containing
+        /// the code is located.</param>
+        /// <param name="s3Key">The S3 key of the zip file.</param>
+        /// <param name="role">A role with the appropriate Amazon Lambda
+        /// permissions.</param>
+        /// <param name="handler">The name of the handler function.</param>
+        /// <returns>The Amazon Resource Name (ARN) of the newly created
+        /// Amazon Lambda function.</returns>
+        public async Task<string> CreateLambdaFunction(
+            AmazonLambdaClient client,
+            string functionName,
+            string s3Bucket,
+            string s3Key,
+            string role,
+            string handler)
+        {
+            var functionCode = new FunctionCode
+            {
+                S3Bucket = s3Bucket,
+                S3Key = s3Key,
+            };
+
+            var createFunctionRequest = new CreateFunctionRequest
+            {
+                FunctionName = functionName,
+                Description = "Created by the Lambda .NET API",
+                Code = functionCode,
+                Handler = handler,
+                Runtime = Runtime.Dotnet6,
+                Role = role,
+            };
+
+            var reponse = await client.CreateFunctionAsync(createFunctionRequest);
+            return reponse.FunctionArn;
+        }
+
+        // snippet-end:[lambda.dotnetv3.Lambda_Basics.CreateLambdaFunction]
+
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.UpdateLambdaConfiguration]
```

Comment:
I do not see this snippet in the metadata yaml.

Reply:
Added.